### PR TITLE
perf(toggleterm): on-demand lazy load

### PR DESCRIPTION
**Pull request by opalmay**
Set keybinds in `init`
Load by any of them

### CODE REVIEW

File: lua/lvim/plugins.lua [hunk 1]
```diff
@@ -265,11 +265,15 @@ local core_plugins = {
...
     "akinsho/toggleterm.nvim",
-    event = "VeryLazy",
     branch = "main",
+    init = function()
+      require("lvim.core.terminal").init()
+    end,
     config = function()
       require("lvim.core.terminal").setup()
     end,
+    keys = lvim.builtin.terminal.keys,
```
Comment (LostNeophyte) [inline lua/lvim/plugins.lua:275]: where is `lvim.builtin.terminal.keys` set?

Comment (opalmay) [reply lua/lvim/plugins.lua]: My bad, fixed

**Comment by opalmay** [comment]
Note that after my planned execs refactor, the logic for init will change. It'll only set the keymaps without any further logic.

File: lua/lvim/plugins.lua [hunk 2]
```diff
@@ -265,11 +265,23 @@ local core_plugins = {
...
+    cmd = {
+      "ToggleTerm",
+      "TermExec",
+      "ToggleTermToggleAll",
+      "ToggleTermSendCurrentLine",
+      "ToggleTermSendVisualLines",
+      "ToggleTermSendVisualSelection",
+    },
+    keys = lvim.builtin.terminal.open_mapping,
+    lazy = true,
```
Comment (LostNeophyte) [inline lua/lvim/plugins.lua:284]:
```suggestion
```
no need to set lazy to true if cmd/keys is set

**Review by LostNeophyte — APPROVED** [review]
LGTM

**Pull request merged**